### PR TITLE
Schedule special weekly build for Thu 19 Dec 2024

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Schedule special weekly build for Thu 19 Dec 2024

Run this build so that we have a Jenkins weekly that can be used in the Jenkins plugin BOM release before the two week break at the end of the year.

Needs approval from @timja before we merge.

### CODE REVIEW

**Review by dduportal — APPROVED** [review]
LGTM on infra side. Waiting for @timja approval

File: Jenkinsfile.d/core/weekly
Comment (timja) [inline Jenkinsfile.d/core/weekly:10]: I don't think doing this will actually do anything, the build has to be run once for the cron trigger to be loaded.

Just trigger it manually

**Review by timja — APPROVED** [review]
Approved for the release but I think this PR is not useful

File: Jenkinsfile.d/core/weekly
Comment (MarkEWaite) [reply Jenkinsfile.d/core/weekly:10]: Thanks!  I hadn't considered that.  Agreed that we'll trigger it manually rather than change the cron job and run it twice.

**Comment by MarkEWaite** [comment]
Closing this pull request because we'll trigger the build manually rather than merging this change and running it once to update the cron then running it again.

**Pull request closed without merging**
